File: 1365. How Many Numbers Are Smaller Than the Current Number/Solution.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        temp = sorted(nums)
        d = {}
        out = []

        for i, val in enumerate(temp):
            if val not in d:
                d[val] = i
        
        for num in nums:
            out.append(d[num])

        return out                

File: 1365. How Many Numbers Are Smaller Than the Current Number/Solution.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] out = new int[nums.length];
        for(int i = 0 ; i<nums.length ; i++){
            int count =0;
            for(int j = 0 ; j<nums.length ; j++){
                if(nums[j] < nums[i]){
                    count++;
                }
            }
            out[i] = count;
        }
        return out;    
    }
}